src/_stylesheets/_prose.scss

```diff
@@ -0,0 +1,37 @@
+.app-prose img,
+.app-prose video {
+  max-width: 100%;
+  height: auto;
+}
+
+.app-prose figure {
+  margin: 0;
+}
+
+.app-prose figcaption {
+  @include govuk-typography-common;
+
+  font-size: 16px;
+  text-align: center;
+}
```

Comment:
Figures aren't something we've strictly decided on using yet (the plan was to suss them out and shortcode them, as in #54), so not sure if want to keep this around or not. 

Could be that it stays in the interim, just be aware that this may move out of `.app-prose` scope at some point.

Reply:
What do you reckon to leaving it for now for the record and scooping it out as part of #54?